egs2/chime4/enh_asr1/conf/tuning/train_enh_asr_wpd_init_noenhloss_wavlm_conformer.yaml

```diff
@@ -0,0 +1,154 @@
+# minibatch related
+batch_type: folded
+batch_size: 16  # A6000 x 1
+accum_grad: 2
+grad_clip: 1
+max_epoch: 31
+patience: 10
+# The initialization method for model parameters
+init: xavier_uniform
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - acc
+    - max
+-   - train
+    - loss
+    - min
+keep_nbest_models: 10
+num_att_plot: 3
+unused_parameters: true
+freeze_param: [
+    "s2t_model.frontend.upstream",
+]
+init_param: [
+    "../enh1/exp/enh_train_enh_beamformer_wpd_ci_sdr_shorttap_raw/valid.loss.best.pth:separator:enh_model.separator",
+    "../asr1/exp/asr_train_asr_conformer_wavlm2_raw_en_char/valid.acc.best.pth:frontend:s2t_model.frontend",
+    "../asr1/exp/asr_train_asr_conformer_wavlm2_raw_en_char/valid.acc.best.pth:preencoder:s2t_model.preencoder",
+    "../asr1/exp/asr_train_asr_conformer_wavlm2_raw_en_char/valid.acc.best.pth:encoder:s2t_model.encoder",
+    "../asr1/exp/asr_train_asr_conformer_wavlm2_raw_en_char/valid.acc.best.pth:ctc:s2t_model.ctc",
+    "../asr1/exp/asr_train_asr_conformer_wavlm2_raw_en_char/valid.acc.best.pth:decoder:s2t_model.decoder",
```

Comment:
Could you also add configurations to obtain the models used here for initialization?

Reply:
It would be nice if you also add some notes in README.md to show how to reproduce the experiments.

Reply:
It is similar to @Emrys365.
Can you provide a specific command (e.g., `local/run_miris.sh`) to reproduce the result, including pre-trained models (specifying a model or training a model)?

Reply:
The pre-training configs specified in this config file (`train_asr_conformer_wavlm2.yaml` and t`rain_enh_beamformer_wpd_ci_sdr_shorttap.yaml`) are included in this pull request.
You mean an additional script, which performs the pre-training of SE and ASR models, would be helpful, right?

Reply:
> You mean an additional script, which performs the pre-training of SE and ASR models, would be helpful, right?

Yes!
Let's make `local/run_miris.sh` and also add a comment to `README.md` on how to train this model from scratch.